### PR TITLE
Helm test

### PR DESCRIPTION
**Pull request by cdoron**
The test deploys the airbyte module pod and service.
It then deploys a client pod, and uses it to obtain
the 'letter_frequency' dataset.

solves #9

### CODE REVIEW

File: helm/README.md [hunk 1]
```diff
@@ -13,8 +13,7 @@
     ./deploy_airbyte_module_client_pod.sh
     ```
 
-    This command deploys a client pod. You get a `bash` prompt, from which you can run:
+    Now that the client pod is deployed, you can obtain the `letter_frequency` dataset from the airbyte-module server by running:
```
Comment (simanadler) [inline helm/README.md:16]: Perhaps add an explanation of the test being performed: dataset used, what the specific airbyte module does, and the expected output.

File: tests/test_helm.sh [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    exit 1
+fi
+
+
+helm install airbyte-module -f helm/abm/values.sample.yaml helm/abm -n default --wait
+
+# create client pod
+cd helm/client
+./deploy_airbyte_module_client_pod.sh
+sleep 5
```
Comment (Mohammad-nassar10) [inline tests/test_helm.sh:40]: It is better to add a while loop for waiting the pod.

File: tests/test_helm.sh [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    echo "test succeeded"
+else
+    RES=1
+fi
+
+if [ ${RES} == 1 ]
+then
+  echo "test failed"
+  exit 1
+fi
```
Comment (Mohammad-nassar10) [inline tests/test_helm.sh:65]: If you deleted the resources before checking the `DIFF`, you can remove the using of `RES`:
```
if [ "${DIFF}" == "" ]
then
    echo "test succeeded"
else
    echo "test failed"
    exit 1
fi

```

File: tests/test_helm.sh (hunk 2 above)
Comment (cdoron) [reply tests/test_helm.sh:40]: I replaced the sleep with a
`kubectl wait pod --for=condition=ready my-shell -n default`
in the `deploy_airbyte_module_client_pod.sh` script

**Review by Mohammad-nassar10 — APPROVED** [review]

**Pull request merged**